### PR TITLE
Optional --skip-email flag for 'wp user update'

### PR DESCRIPTION
**Pull request by JanVoracek**
Sometimes (e.g. in our use case) it might be handy to update the user's details without bothering him with an email.

I would write some tests but I'm not really sure how to test it. I was looking for tests of `wp core install --skip-email` and they are missing as well.

### CODE REVIEW

**Comment by danielbachhuber** [comment]
> I would write some tests but I'm not really sure how to test it.

Here's an example: https://github.com/wp-cli/entity-command/blob/4933b26dc5f64631a85997281a25bbb6e397e42f/features/user-reset-password.feature#L16

File: src/User_Command.php [hunk 1]
```diff
@@ -498,6 +501,11 @@ public function update( $args, $assoc_args ) {
 			$user_ids[] = $user->ID;
 		}
 
+		if ( isset( $assoc_args['skip-email'] ) ) {
```
Comment (danielbachhuber) [inline src/User_Command.php]: This should use `WP_CLI\Utils\get_flag_value()`

File: src/User_Command.php [hunk 2]
```diff
@@ -498,6 +501,11 @@ public function update( $args, $assoc_args ) {
...
+			add_filter( 'send_email_change_email', '__return_false' );
+			add_filter( 'send_password_change_email', '__return_false' );
```
Comment (danielbachhuber) [inline src/User_Command.php]: Can we remove these filters after the fact, to ensure they don't bleed through to other commands?

**Comment by danielbachhuber** [comment]
@JanVoracek Still planning to come around and fix this up?

**Comment by JanVoracek** [comment]
Yes, I will fix it. Sorry for waiting.

**Comment by schlessera** [comment]
@JanVoracek Are you still up for working on this? Do you need any further help with something?

**Comment by schlessera** [comment]
Thanks for your work on this, @JanVoracek !

**Pull request merged**